### PR TITLE
Added DB configuration to backend app.

### LINKED ISSUE #70: Add postgres to backend app
and postgres a proof of concept patient and/or device placeholder table

### PR DESCRIPTION
**Pull request by benwarfield-usds**
Contains:
* Spring Data JPA  and liquibase/hibernate dependencies
* Updated docker-compose configuration 
* _de minimis_  liquibase configuration
* updated application configuration, sadly now requiring use of a `dev` profile for local development

Resolves #70.

### CODE REVIEW

File: backend/build.gradle [hunk 1]
```diff
@@ -13,9 +13,18 @@ repositories {
 }
 
 dependencies {
+	// core infrastructure
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+
+	// data layer dependencies
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation "com.vladmihalcea:hibernate-types-52:2.5.0"
+	implementation "org.liquibase:liquibase-core"
```
Comment (nickrobison-usds) [inline backend/build.gradle:22]: Are these versions specified in the dependency management pom?

**Review by nickrobison-usds — COMMENTED** [review]
Nice work!

File: backend/build.gradle (hunk 1 above)
Comment (benwarfield-usds) [reply backend/build.gradle:22]: Yep! Or I should say, by way of the dependency management plugin, through a process that I do not need to know the internals of, so I don't. 😬

File: backend/Dockerfile.prod [hunk 2]
```diff
@@ -1,9 +1,9 @@
-FROM gradle:6.7.0-jdk8-hotspot AS build
+FROM gradle:6.6.1-jdk8-hotspot
```
Comment (benwarfield-usds) [inline backend/Dockerfile.prod:1]: Tsk!

File: backend/build.gradle [hunk 3]
```diff
@@ -13,9 +13,18 @@ repositories {
 }
 
 dependencies {
+	// core infrastructure
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+
+	// data layer dependencies
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation "com.vladmihalcea:hibernate-types-52:2.5.0"
```
Comment (benwarfield-usds) [inline backend/build.gradle:21]: Bump to latest.

**Pull request merged**